### PR TITLE
CD-207: Proxmox VMs List (table)

### PR DESCRIPTION
**Pull request by mariemm97**

### CODE REVIEW

File: examples/ssh/proxmox_vm_list.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            "hypervisor"
+        ]
+    },
+    "tags": [
+        "proxmox",
+        "vm",
+        "qm-list",
+        "qm-config"
+    ],
+    "sample_period": 300,
```
Comment (kgskgs) [inline examples/ssh/proxmox_vm_list.json:22]: change to sample_period_s

**Review by kgskgs — APPROVED** [review]

File: examples/ssh/proxmox_vm_list.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    password: D.device.password(),
+    timeout: 120000
+};
+ 
+var kvmsTable = D.createTable(
+    "Virtual Machines List",
+    [
+        { label: "Name" },
+        { label: "Status" },
+        { label: "Memory", unit: "MB" },
```
Comment (kgskgs) [inline examples/ssh/proxmox_vm_list.js:36]: shouldn't this and the below have valueType NUMBER ?

**Review by kgskgs — APPROVED** [review]

File: examples/ssh/proxmox_vm_list.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+ *      - ostype : Operating system type
+ *      - Error(s) : Errors in getting info on that specific VM
+ * 
+ **/
+
+var cmdKVMsList = "qm list";
+var sshConfig = {
+    username: D.device.username(),
+    password: D.device.password(),
+    timeout: 120000
```
Comment (GeorgiTsatsev) [inline examples/ssh/proxmox_vm_list.js:28]: timeout is too high for a single SSH command

File: examples/ssh/proxmox_vm_list.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            "hypervisor"
+        ]
+    },
+    "tags": [
+        "proxmox",
+        "vm",
+        "qm-list",
+        "qm-config"
+    ],
+    "sample_period_s": 300,
```
Comment (GeorgiTsatsev) [inline examples/ssh/proxmox_vm_list.json:22]: maybe higher than 5 min

File: examples/ssh/proxmox_vm_list.js [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+/**
+  * @remote_procedure
+  * @label Validate Association
+  * @documentation This procedure is used to validate if the SSH command is executed successfully
+  */
+function validate() {
+    executeCommand(cmdKVMsList)
+        .then(function (data) {
+            if (data) {
```
Comment (GeorgiTsatsev) [inline examples/ssh/proxmox_vm_list.js:164]: please validate properly if the command was OK

**Review by GeorgiTsatsev — APPROVED** [review]

**Pull request merged**